### PR TITLE
Signet Ring Quest Reward Overhaul

### PR DESCRIPTION
**Pull request by FSU-Phil**
This change is an overhaul of the Signet Ring Quest Reward table:

When you return a signet ring your chances (of getting 1 of below) are:
40% = Money (20-40k)
15% = Invis Ring + Money (5-10k)
15% = (25 Purps, or 25 Ents) + Money (5-10k)
10% = Rose
10% = (25 Inkies, or 25 Mints) + Money (5-10k)
5% = Random Sided Prism
5% = Random Mask Gift

### CODE REVIEW

**Comment by Koveras** [comment]
+1

File: kod/object/item/passitem/ring/ringsgnt.kod [hunk 1]
```diff
@@ -91,26 +94,150 @@ messages:
...
+         oItem1 = Create(&SpecialGift);
+         Send(who,@NewHold,#what=oItem1);
+
+         return;
       }
+      
       else
       {
-         Send(oMoney,@AddNumber,#number=iValue);
+         Debug("Signet Ring Reward did not meet criteria for ",who);
```
[the remark this answers is not in the document]
Comment (davidklaw) [reply kod/object/item/passitem/ring/ringsgnt.kod:239]: @FSU-Phil Still around? 🤔 Any interest in following up on this?

**Comment by davidklaw** [comment]
@FSU-Phil Still around? 🤔 Any interest in following up on this?

**Comment by davidklaw** [comment]
`trestype.kod` still limits signet ring drops to players under the `PFLAG_PKILL_ENABLE` threshold. Is it odd to be handing out these types of rewards to new characters? Anything to make the signet ring quests more interesting or valuable under the current game economy is welcome in my opinion, but this seems a little extreme.

What about something like the reward table below? These seem like worthwhile rewards to support a new player or provide them with something unique and interesting to hang onto.

| Probability | Reward | 
| --- | ----------- |
| 90% | 2,500-7,500 shillings |
| 5% | Plate armor |
| 3% | Jewel of Froz |
| 1% | Rose |
| 1% | Random mask gift |

Adding dialog for these could be fun as well.

**Comment by davidklaw** [comment]
If the original intent behind this PR is desired and @FSU-Phil is no longer around, I'd be happy to address the last request and submit a new one.

**Comment by akirmse** [comment]
Closing this, as no motivation is given for the change.  We could start over if there's a need for a change,

**Pull request closed without merging**